client/js/monthData.js

```diff
@@ -0,0 +1,151 @@
+/*Temporary Mock Month Data */
+module.exports = {
+  'week1': [
+    {
+      date: '05012015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05022015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05032015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05042015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05052015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05062015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date:'05072015',
+      numEvents: 3,
+      status: 'good'
+    }
+  ],
+  'week2': [
+    {
+      date: '05082015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05092015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05102015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05112015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05122015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05132015',
+      numEvents: 3,
+      status: 'good'
+    },
+    {
+      date: '05142015',
+      numEvents: 3,
+      status: 'good'
+    }
+  ],
+  'week3': [
+    {
+      date: '05152015',
+      numEvents: 3,
+      status: 'bad'
+    },
+    {
+      date: '05162015',
+      numEvents: 3,
+      status: 'bad'
+    },
+    {
+      date: '05172015',
+      numEvents: 3,
+      status: 'bad'
+    },
+    {
+      date: '05182015',
+      numEvents: 3,
+      status: 'bad'
+    },
+    {
+      date: '05192015',
+      numEvents: 3,
+      status: 'bad'
+    },
+    {
+      date: '05202015',
+      numEvents: 3,
+      status: 'bad'
+    },
+    {
+      date: '05212015',
+      numEvents: 3,
+      status: 'bad'
+    },
```

Comment:
Line 112. You represent a week with an array of objects and the last object in the array has a comma after it. Same problem below.